syntax = "proto3";

service Grader {
  rpc grade (GradingRequest) returns (GradingResponse) {};
}

message GradingRequest {
  int64 exam_content_id = 1;
  bool grade_type = 2;
}

message GradingResponse {
  map<int64, float> final_score = 1;
}